examples/ota-requestor-app/linux/main.cpp

```diff
@@ -54,23 +54,22 @@ OTAImageProcessorImpl gImageProcessor;
 bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier, const char * aName, const char * aValue);
 void OnStartDelayTimerHandler(Layer * systemLayer, void * appState);
 
-constexpr uint16_t kOptionDelayQuery          = 'q';
-constexpr uint16_t kOptionRequestorCanConsent = 'c';
+constexpr uint16_t kOptionPeriodicQueryTimeout = 'p';
+constexpr uint16_t kOptionRequestorCanConsent  = 'c';
 
-uint16_t delayQueryTimeInSec = 0;
+uint32_t gPeriodicQueryTimeout = (24 * 60 * 60);
```

Comment:
Would prefer if units were specified in the variable name to help code readability

Reply:
Fixed